sample-app/src/main/java/com/joinforage/android/example/ui/pos/POSViewModel.kt

```diff
@@ -136,9 +139,10 @@ class POSViewModel : ViewModel() {
         }
     }
 
-    fun tokenizeEBTCard(foragePanEditText: ForagePANEditText, terminalId: String, onSuccess: (data: PosPaymentMethod?) -> Unit) {
+    fun tokenizeEBTCard(context: Context, foragePanEditText: ForagePANEditText, terminalId: String, onSuccess: (data: PosPaymentMethod?) -> Unit) {
```

Comment:
The same sample-app changes have already been reflected in the private repo distribution